deezerapp/src/app/services/artists.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { of, Observable } from "rxjs";
+import { environment } from "src/environments/environment";
+import { Artist } from "../models/artist";
+import { InjectableCompiler } from "@angular/compiler/src/injectable_compiler";
+import { tap, catchError } from "rxjs/operators";
+
+@Injectable({
+  providedIn: "root"
+})
+export class ArtistsService {
+  constructor(private http: HttpClient) {}
+
+  getArtists(query?: number): Observable<Artist> {
+    return this.http.get<Artist>(`${environment.apiUri}/artist/${query}`);
+  }
+
+  searchArtists(term?: string): Observable<Artist> {
```

Comment:
`searchArtists` should return a collection of artists? i.e. `Observable<Aritist[]>`.